sklearn/_config.py

```diff
@@ -43,11 +45,21 @@ def set_config(assume_finite=None, working_memory=None):
 
         .. versionadded:: 0.20
 
+    print_changed_only : bool, optional
```

Comment:
would this be clearer as print_defaults?

Reply:
I don't think so because the meaning of the parameter is really "only show the parameters that don't have their default values, i.e. the ones that are changed"

"print_defaults" would mean something quite different.

Maybe "hide_defaults"? Or "print_hide_defaults"?